shared/src/ledger/eth_bridge/vp/authorize.rs

```diff
@@ -0,0 +1,36 @@
+//! Functionality to do with checking whether a transaction is authorized by the
+//! "owner" of some key under this account
+use eyre::Result;
+
+use super::store;
+use crate::types::address::Address;
+
+pub(super) fn is_authorized(
```

Comment:
I'm trying to follow the idea here. Is that idea that since the users vps won't be triggered otherwise, we need to manually trigger them to see if they approve a multitoken transfer? If so, shouldn't we pass in the verifiers set here and insert the necessary vps?

Reply:
> Implicit VPs are required for Namada launch, so I think triggering the necessary sender and receiver vps manually in this vp is the way to go.

Implicit VP issue - https://github.com/anoma/namada/issues/8

I will scope out https://github.com/anoma/namada/issues/432 further. There are some issues I think with the implicit VP approach in general, not directly related to the eth bridge, I can write up. For now then we can at least get parity with what works for normal tokens (i.e. proper authorization for transfers sent from an established address). Authorization for transfers sent from implicit addresses does not currently work even for normal token transfers using `namadac transfer`, we will need to wait for https://github.com/anoma/namada/issues/8 to be implemented